### PR TITLE
convenience methods and java version

### PR DESCRIPTION
**Pull request by lburgazzoli**
Includes code for:
- convenient method #16 
- java version #5

### CODE REVIEW

File: src/main/java/com/coreos/jetcd/EtcdHelper.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.coreos.jetcd;
```
Comment (xiang90) [inline src/main/java/com/coreos/jetcd/EtcdHelper.java]: remove this file for now? nothing uses it.

**Comment by xiang90** [comment]
other than that, lgtm

**Comment by lburgazzoli** [comment]
done

**Comment by xiang90** [comment]
thanks.

**Pull request merged**